lxd/project/permissions.go

```diff
@@ -152,22 +152,38 @@ func checkRestrictionsOnVolatileConfig(project *api.Project, instanceType instan
 		return nil
 	}
 
+	// List of safe keys
+	safe_keys := [5]string{"volatile.apply_template", "volatile.base_image", "volatile.last_state.power", "volatile.DEVNAME.apply_quota", "volatile.DEVNAME.hwaddr"}
```

Comment:
So in @stgraber original comment https://github.com/lxc/lxd/issues/7896#issue-705113165 the `DEVNAME` bit is dynamic and represents the name of the instance device.

So for example `volatile.eth0.hwaddr` or `volatile.eth1.hwaddr`, rather than the literal `DEVNAME` matching.